tl/cli/feature-voting.py

```diff
@@ -0,0 +1,68 @@
+import sys
+import traceback
+import argparse
+
+from tl.exceptions import TLException
+
+
+def parser():
+    return {
+        'help': """
+        Score candidates using feature voting. List of features may be used: 
+        1. exact match top 1: using pre-computed embedding vectors, either from a file or from elasticsearch.
+        2. page rank 
+        3. qnode with smallest number
+        4. Weiss Waterman distance
+        5. Jaccard between description and row
+        6. column query (SemTab 2020 paper)
```

Comment:
Where are these options in the command line interface ? In other words, how can user pick and choose between these options?